Scuti/Scripts/UI/Store/OffersPresenterPortrait.cs

```diff
@@ -152,21 +152,23 @@ private async void OnPresenterClicked(OfferSummaryPresenterBase presenter)
                 UIManager.ShowLoading(false);
                 var id = presenter.Data.ID;
                 var offer = await ScutiNetClient.Instance.Offer.GetOfferByID(id);
-                var panelModel = Mappers.GetOfferDetailsPresenterModel(offer);
-
-                try
-                {
-                    UIManager.OfferDetails.SetData(panelModel);
-                    UIManager.OfferDetails.SetIsVideo(!string.IsNullOrEmpty(presenter.Data.VideoURL));
-                    UIManager.Open(UIManager.OfferDetails);
-                }
-                catch (Exception e)
+                if (!ScutiUtils.HasLink(offer))
                 {
-                    ScutiLogger.LogException(e);
-                    UIManager.Alert.SetHeader("Out of Stock").SetBody("This item is out of stock. Please try again later.").SetButtonText("OK").Show(() => { });
-                    //UIManager.Open(UIManager.Offers);
+                    var panelModel = Mappers.GetOfferDetailsPresenterModel(offer);
+
+                    try
+                    {
+                        UIManager.OfferDetails.SetData(panelModel);
+                        UIManager.OfferDetails.SetIsVideo(!string.IsNullOrEmpty(presenter.Data.VideoURL));
+                        UIManager.Open(UIManager.OfferDetails);
+                    }
+                    catch (Exception e)
+                    {
+                        ScutiLogger.LogException(e);
+                        UIManager.Alert.SetHeader("Out of Stock").SetBody("This item is out of stock. Please try again later.").SetButtonText("OK").Show(() => { });
+                        //UIManager.Open(UIManager.Offers);
+                    }
                 }
```

Comment:
Record offer interaction here similar to banner widget.  Possibly move to HasLink fucntion instead of doing this in 3 places..